src/invoices/Voucher/VoucherEditForm/VoucherEditForm.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+
+import stripesFinalForm from '@folio/stripes/final-form';
+import {
+  Row,
+  Col,
+  TextField,
+  Pane,
+  Paneset,
+} from '@folio/stripes/components';
+import {
+  FieldDatepickerFinal as Datepicker,
+  FormFooter,
+} from '@folio/stripes-acq-components';
+import {
+  AppIcon,
+} from '@folio/stripes/core';
+
+const VoucherEditForm = ({
+  handleSubmit,
+  submitting,
+  pristine,
+  onCancel,
+  initialValues,
+  vendorInvoiceNo,
+  isAllowVoucherNumberEdit,
+}) => {
+  const paneFooter = (
+    <FormFooter
+      id="clickable-save"
+      label={<FormattedMessage id="ui-invoice.saveAndClose" />}
+      pristine={pristine}
+      submitting={submitting}
+      handleSubmit={handleSubmit}
+      onCancel={onCancel}
+    />
+  );
+
+  return (
+    <Paneset
+      data-test-edit-voucher-form
+    >
+      <Pane
+        appIcon={<AppIcon app="invoice" size="small" />}
+        defaultWidth="fill"
+        dismissible
+        id="pane-edit-voucher"
+        footer={paneFooter}
+        onClose={onCancel}
+        paneSub={
+          <FormattedMessage
+            id="ui-invoice.voucher.paneSubTitle"
+            values={{ voucherNumber: initialValues.voucherNumber }}
+          />
+        }
+        paneTitle={
+          <FormattedMessage
+            id="ui-invoice.voucher.paneTitle"
+            values={{ vendorInvoiceNo }}
+          />
+        }
+      >
+        <Row>
+          <Col xs={3}>
+            <Field
+              component={TextField}
+              label={<FormattedMessage id="ui-invoice.invoice.details.voucher.voucherNumber" />}
+              name="voucherNumber"
+              disabled={!isAllowVoucherNumberEdit}
+            />
+          </Col>
+          <Col xs={3}>
+            <Field
+              component={TextField}
+              label={<FormattedMessage id="ui-invoice.invoice.details.voucher.disbursementNumber" />}
+              name="disbursementNumber"
+            />
+          </Col>
+          <Col xs={3}>
+            <Field
+              component={Datepicker}

```

Comment:
this looks wrong. You can check other usages:
https://github.com/folio-org/ui-receiving/blob/cbe38d63089b56e7e66340318adb4320089db60c/src/TitleForm/TitleForm.js#L165
https://github.com/folio-org/ui-receiving/blob/cd36ae8297b5495aef3b847784b8356de6ad0deb/src/TitleDetails/AddPieceModal/AddPieceModal.js#L135